src/Tizen.NUI/src/public/XamlBinding/ContentPage.cs

```diff
@@ -110,6 +110,7 @@ public ContentPage(Window win)
             Root.HeightResizePolicy = ResizePolicyType.FillToParent;
 
             win.Add(Root);
+            win.AddPage(this);
```

Comment:
Because of this part, creating a new instance of the same type will trow an exception. is it intended?